packages/react-components/src/Tabs/index.tsx

```diff
@@ -107,6 +108,7 @@ function Tabs ({ basePath, className = '', hidden, isSequence, items }: Props):
           </li>
         ))}
       </ul>
+      <HelpOverlay/>
```

Comment:
Would be happy dropping this here. 

It is a good idea, but actually so outdated info-wise that it is probably dangerous. 

(There is one exception - the developer tab in settings. But there can find another solution if dropped)